contents/teams/content/objectives.mdx

```diff
@@ -82,179 +131,304 @@
 ## Docs
```

Comment:
From past experience, I suspect y'all have 2 too many goals. Do with this information what you will 🤷